css/selectors/where.json

```diff
@@ -7,7 +7,14 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/:where",
           "support": {
             "chrome": {
-              "version_added": false
+              "version_added": "83",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "Experimental Web Platform Features",
+                  "value_to_set": "enabled"
+                }
+              ]
             },
             "chrome_android": {
               "version_added": false
```

Comment:
Can you duplicate the data point above to Chrome for Android? They're nearly always the same

Reply:
If they're not, that will be reflected in the Chrome Status entry.